.github/workflows/test_definitions.yml

```diff
@@ -6,53 +6,56 @@ on:
   workflow_call:
     # Inputs and secrets documented in the caller workflows
     inputs:
-      clang-tidy:
+      ros-ci:
         required: true
         type: boolean
-      custom-interfaces:
+      clang-tidy:
         required: true
         type: boolean
-      disable-vcs:
-        required: true
-        type: string
       repository:
         required: true
         type: string
       rebuild-docs:
         required: true
         type: boolean
+      markdown-ci:
+        required: false
+        default: true
+        type: boolean
     secrets:
       gh-token:
         required: true
 
 jobs:
   colcon-test:
     runs-on: ubuntu-latest
+    if: ${{ inputs.ros-ci }}
     steps:
-      - name: Checkout workspace
-        uses: actions/checkout@v3
-        with:
-            repository: UBCSailbot/sailbot_workspace
-
-      - name: Checkout ROS package
-        if: ${{ inputs.repository }} != ${{ github.event.repository.name }}
-        uses: actions/checkout@v3
-        with:
-            repository: UBCSailbot/${{ inputs.repository }}
-            path: src/${{ inputs.repository }}
-
-      - name: Checkout custom_interfaces ROS package
-        if: ${{ inputs.custom-interfaces}}
-        uses: actions/checkout@v3
-        with:
-            repository: UBCSailbot/custom_interfaces
-            ref: user/colinli02/custom_messages
-            path: src/custom_interfaces
-
-      - name: Test
-        uses: ./.github/actions/test/
-        env:
-          DISABLE_VCS: ${{ inputs.disable-vcs }}
+    - name: Checkout workspace
+      uses: actions/checkout@v3
+      with:
+          repository: UBCSailbot/sailbot_workspace
+          ref: user/dfriend01/add_lang_ci_args
+
+    - name: Checkout ROS package
+      if: ${{ inputs.repository != github.event.repository.name }}
+      uses: actions/checkout@v3
+      with:
+          repository: UBCSailbot/${{ inputs.repository }}
+          path: src/${{ inputs.repository }}
+
+    - name: Checkout custom_interfaces ROS package
+      if: ${{ inputs.repository != github.event.repository.name }}
+      uses: actions/checkout@v3
+      with:
+          repository: UBCSailbot/custom_interfaces
+          ref: user/colinli02/custom_messages
+          path: src/custom_interfaces
+
+    - name: Test
+      uses: ./.github/actions/test/
+      env:
+        DISABLE_VCS: ${{ inputs.repository != github.event.repository.name }}
```

Comment:
Did you check that this works when it's true and when it's false?

Reply:
It has been confirmed that bash can accept booleans and can still interpret them as strings in the script.